dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/model/drc/DrcDataRequest.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.laa.crime.dces.integration.model.drc;
```

Comment:
With a class name of DrcDataRequest, does this need to be separated out into it's own package?